ClassySchedule/src/main/java/scheduler/AddCourseToScheduleController.java

```diff
@@ -225,31 +274,219 @@ public LocalDateTime convertToLocalDateTimeViaInstant(Date dateToConvert) {
+                result = true;
+                break;
+            }
+        }
+
+        // boolean outcome is false, meaning none of the days has been selected, so
+        // use JavaFX to prompt the user to select at least 1 day of the week.
+        if(result == false) {
+            invalidDays.showAndWait();
+        }
+
+        return result;
+    }
+
+    /**
+     * This function validate that the user has inputed the start and end time in the correct 24-hour time format
+     * and that end time must be after start time. Otherwise, it will be invalid.
+     * @return false is time is incorrect, else return true
+     */
+    private boolean validateTime() {
+        // TODO: time error showing up twice. Also need to validate that end time must be after start time and that it is an integer that is being entered
+        boolean result = true;
+        String startTime = start_time.getText();
+        int startTimeValue = -1;
+
+        if(startTime.length() == 0) {
+            result = false;
+        }
+
+        if(startTime.length() > 5) {
+            result = false;
+        } else {
+            if(startTime.length() == 5) {
+                if(startTime.charAt(2) != ':') {
+                    StringBuilder sb = new StringBuilder();
+                    sb.append(startTime,0, 2);
+                    startTimeValue = Integer.parseInt(sb.toString());
+                    result = false;
+                } else {
+                    if(startTime.charAt(1) != ':') {
+                        StringBuilder sb = new StringBuilder();
+                        sb.append(startTime, 0, 1);
+                        startTimeValue = Integer.parseInt(sb.toString());
+                        result = false;
+                    }
+                }
+            }
+        }
+
+        String endTime = end_time.getText();
+        int endTimeValue = -1;
+
+        if(endTime.length() == 0) {
+            result = false;
+        }
+
+        if(endTime.length() > 5) {
+            result = false;
+        } else {
+            if(endTime.length() == 5) {
+                if(endTime.charAt(2) != ':') {
+                    StringBuilder sb = new StringBuilder();
+                    sb.append(endTime,0, 2);
+                    endTimeValue = Integer.parseInt(sb.toString());
+                    result = false;
+                } else {
+                    if(endTime.charAt(1) != ':') {
+                        StringBuilder sb = new StringBuilder();
+                        sb.append(endTime, 0, 1);
+                        endTimeValue = Integer.parseInt(sb.toString());
+                        result = false;
+                    }
+                }
+            }
+        }
+
+
+        if(startTimeValue > endTimeValue) {
+            result = false;
+        }
+
+        if(result == false) {
+            invalidStartAndEndTime.showAndWait();
+        }
+
+        return result;
+    }
+
+    // 1:00, 12:00
```

Comment:
If this code is not needed, then it should be deleted and not included in the PR